### ISSUE TITLE
FEATURE - Add Board Source

### ISSUE THREAD

**Issue by KaoruDev** [fivetran/dbt_jira_source#26]
**Are you a Fivetran customer?** 

Yes, Kaoru Kohashigawa, Senior DevOps Engineer @ Binti

**Is your feature request related to a problem? Please describe.**
 
Nope

**Describe the solution you'd like**
 
Be able to use `dbt` `source` macro.

**Describe alternatives you've considered**
 
I'll define the source myself.

**Additional context**

N/A

**Please indicate the level of urgency and business impact of this request** 

non-urgent.

**Are you interested in contributing to this package?** 

- [x] Yes, I can do this and open a PR for your review.
- [ ] Possibly, but I'm not quite sure how to do this. I'd be happy to do a live coding session with someone to get this work implemented. 
- [ ] No, I'd prefer if someone else did this.  I don't have the time and/or don't know how to incorporate the changes necessary.

The thing I'll need help with is finding the definitions of the columns

**Comment by fivetran-joemarkiewicz** [comment]
Hi @KaoruDev thanks so much for opening this feature request.

I would love to get a better understanding of the feature you are wanting the package to include. It looks like you want to be able to run `dbt source-freshness` is that correct? If that is the case, you should be able to accomplish this as our source file contains freshness tests. Further, if you wanted to apply custom freshness tests, then you should be able to override our freshness configs by leveraging the [source override](https://docs.getdbt.com/reference/resource-properties/overrides) functionality.

Let me know if this helps. Otherwise, I would love to understand this feature request better. Thanks! 😄

**Comment by KaoruDev** [comment]
In my dbt project I'd like to create a model by referencing the `board` table, so I'd write `source("jira", "board")` but because the source model does not exist in this file: https://github.com/fivetran/dbt_jira_source/blob/main/models/src_jira.yml  dbt will throw an error.

I got around this by defining the source model in my own dbt project.

**Comment by fivetran-joemarkiewicz** [comment]
Ahhh that makes sense. You are looking to add support for the additional `board` source table to be included in the package. This is definitely something we would be able to add to the package in a future sprint. If this is something you wanted sooner, you can always open a PR to integrate this new model and source. 

One thing I would like to know better is what your intentions are of using the `board` table in downstream models?

**Comment by KaoruDev** [comment]
Non-urgent since I have a relatively okay workaround. I'd be happy to contribute a PR, I think the only thing I'm missing is the description of the columns of the table. Not sure where I could find that? 

Looking at the analyst's query, it appears we want to associate a sprint to a board's name.

**Comment by fivetran-joemarkiewicz** [comment]
That makes complete sense. I will move this feature enhancement to our backlog for the time being and can share an update once we prioritize this update. If you would like to still open the PR then my team and I can take the time to gain an understanding of the column descriptions and apply them before release. 

Let me know if you would like any more guidance if you decide to open a PR.
